monochart/values.yaml

```diff
@@ -324,6 +324,26 @@ service:
   # annotations:
   #   name: value
 
+mapping:
+  normal:
+    enabled: false
+    # host: normal.{{ $AppName }}.{{ .Environment.Name }}.spoton.sh
+    # prefix: /my-prefix
+    # rewrite: /my-rewrite
+    # labels:
+    #   name: value
+    # annotations:
+    #   name: value
+  mygrpc:
```

Comment:
Actually one note, why `mygrp` instead of just `grpc`?

Reply:
I updated them @kos-kamil to `api` and `grpc`. Those sound more standard. Looks like it didn't stale the review, so just letting you know. Thanks.